src/sql/workbench/parts/notebook/test/node/contentManagers.test.ts

```diff
@@ -40,7 +46,23 @@ function verifyMatchesExpectedNotebook(notebook: nb.INotebookContents): void {
 }
 
 suite('Local Content Manager', function (): void {
-	let contentManager = new LocalContentManager();
+	let contentManager: LocalContentManager;
+
+	setup(() => {
```

Comment:
We've verified all tests pass after this?

Reply:
Well these are unit tests so they run in the pipeline.